pom.xml

```diff
@@ -13,7 +13,11 @@
 
   <properties>
     <revision>1.9.0-SNAPSHOT</revision>
-    <debezium.base.version>${revision}</debezium.base.version>
+    <!--
+    For now lets explicitly lock into a specific Debezium version
+    Do not forget to adjust .github/workflows/end-to-end-testing.yaml to align with this version
+    -->
+    <debezium.base.version>2.0.0.Alpha3</debezium.base.version>
```

Comment:
This should be done only after @ani-sha's PR to update to 1.9.4. Also I think updating the `$revision` is the right way to do the upgrade or is there a reason to only update the base version?

Reply:
@rk3rn3r Should I open my PR first or should we create the branches first? I am confused about that part.

Reply:
@ani-sha I would personally do all 1.9 related work on the main branch first. Then branch out a 1.9 branch. This way we don't need to merge back 1.9 branch commits to the main branch, resulting in cleaner history imo and it might be easier for other contributors. But if you prefer another way that's also doable.

Reply:
As I commented on the 1.9.5.Final PR, we need to expressly use the `debezium/connect:1.9` branch for that work and we need to address any concerns about whether all the bits for the REST extension are there, etc.  It makes no sense to forward the unit tests and the source to 1.9.5.Final but to perform integration tests against 2.0.